dev-util/cgvg/cgvg-1.6.3.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A tiny version of cscope that is much more useful in certian instances"
```

Comment:
spot the typo! also, it may be more helpful to tell the certain instances rather than copy the text from the advert, but I realise this is not coming from you.